src/api/market.ts

```diff
@@ -33,3 +34,22 @@ export const getProduct = async (
   );
   return response.data;
 };
+
+export const postProduct = async (
+  token: string,
+  formData: ProductFormValues,
+  categoryId: number | undefined,
+) => {
+  if (categoryId === 0) {
+    throw new Error('카테고리를 선택해주세요.');
+  }
+  await Axios.post(
+    `/api/categories/${categoryId}/products`,
+    {
+      title: formData.title,
+      price: formData.price,
+      content: formData.content,
+    },
+    { headers: { Authorization: `Bearer ${token}` } },
+  );
```

Comment:
아니 이럴거면 그냥

```suggestion
  await Axios.post(
    `/api/categories/${categoryId}/products`,
    formData,
    { headers: { Authorization: `Bearer ${token}` } },
  );
```
이러면 되는거 아니야..? ㅋㅋㅋㅋ

Reply:
ㅎㅎ.........수정 완완입니다 ^_^7